src/chain.cpp

```diff
@@ -4,13 +4,21 @@
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
 #include <chain.h>
+#include <tinyformat.h>
 #include <util/time.h>
 
 std::string CBlockFileInfo::ToString() const
 {
     return strprintf("CBlockFileInfo(blocks=%u, size=%u, heights=%u...%u, time=%s...%s)", nBlocks, nSize, nHeightFirst, nHeightLast, FormatISO8601Date(nTimeFirst), FormatISO8601Date(nTimeLast));
 }
 
+std::string CBlockIndex::ToString() const
+{
+    assert(phashBlock != nullptr);
```

Comment:
no need to call the assert twice in this function?

Reply:
Right, the first commit bc7d14164b `CBlockIndex: ensure phashBlock is not nullptr before dereferencing` seems unnecessary.

Reply:
Indeed. Done.